index.d.ts

```diff
@@ -39,13 +39,14 @@ export function getSettings(workspace: ProjectWorkspace, options?: Options): Pro
 export class ProjectWorkspace {
   constructor(
     rootPath: string,
-    pathToJest: string,
+    jestCommandLine: string,
     pathToConfig: string,
-    localJestMajorVersin: number,
+    localJestMajorVersion: number,
     outputFileSuffix?: string,
     collectCoverage?: boolean,
     debug?: boolean,
   );
+  jestCommandLine: string;
   pathToJest: string;

```

Comment:
given this is a type file that it won't impact runtime (the binary out there that uses pathToJest will continue to work), we could get rid of `pathToJest`, so our customers will see ts warning during development and will need to fix the signature in their next release.